maintainers/maintainer-list.nix

```diff
@@ -15870,6 +15870,15 @@
     githubId = 1312525;
     name = "Rongcui Dong";
   };
+  rookeur = {
```

Comment:
Split

Reply:
@AndersonTorres if you are taking the effort to review pull requests, please do it properly and at the very least take the time to write a proper sentence.

@Rookeur adding yourself as maintainer needs to be split into its own commit. This has not been done yet, so I've unresolved this thread again.

Reply:
@piegamesde sorry for the misunderstanding i have split the commit

Reply:
No worries. Tbh, the misunderstanding was not your fault.